### ISSUE TITLE
AssessmentResults findings with flexible results values

### ISSUE THREAD

**Issue by degenaro** [usnistgov/OSCAL#841]
# User Story:

As an OSCAL tool developer, I would like AssessmentResults `findings` to have results values beyond the `target.status` allowed values `{ satisfied, not satisfied }`.

## Goals:

In earlier versions of the OSCAL schema, each `finding` had
 ```
    result
          STRVALUE: <value>
          system: <system>
```

This was desirable since valid values were flexible in accordance with a specified system. To align with _XCCDF_, for example, see [xccdf spec](https://csrc.nist.gov/CSRC/media/Publications/nistir/7275/rev-4/final/documents/nistir-7275r4_updated-march-2012_clean.pdf) Table 26. Possible `STRVALUE` values: `{ pass, fail, error, unknown, notapplicable, notchecked, notselected, informational, fixed }`.

`` 
## Dependencies:

None.

## Acceptance Criteria

- [x] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [x] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [ ] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

**Comment by david-waltermire-nist** [comment]
Can you provide definitions for: pass, fail, error, unknown, notapplicable, notchecked, notselected, informational, fixed?

**Comment by degenaro** [comment]
re: definitions, see Table 26 on page 43 of [xccdf spec](https://csrc.nist.gov/CSRC/media/Publications/nistir/7275/rev-4/final/documents/nistir-7275r4_updated-march-2012_clean.pdf).
**Specification for the Extensible Configuration Checklist Description Format (XCCDF)Version 1.2** 
David Waltermire
Charles Schmidt
Karen Scarfone
Neal Ziring.

**Comment by david-waltermire-nist** [comment]
@stephenbanghart I'd like you to organize this as a model review discussion topic.

**Comment by david-waltermire-nist** [comment]
This is addressed by PR #944 based on the conversations we have had during the OSCAL model review.

**Comment by david-waltermire-nist** [comment]
Addressed by PR #951.
